backpack/core/derivatives/conv_transposend.py

```diff
@@ -77,9 +77,6 @@ def _weight_jac_mat_prod(self, module, g_inp, g_out, mat):
         return self.reshape_like_output(jac_mat, module)
 
     def _weight_jac_t_mat_prod(self, module, g_inp, g_out, mat, sum_batch=True):
```

Comment:
Please add a docstring with the standards specified in https://github.com/f-dangel/backpack/blob/development/README-dev.md#pull-requests